R/dependencies.R

```diff
@@ -0,0 +1,23 @@
+css_dependency <- function(name, files) {
+  htmltools::htmlDependency(
+    name,
+    utils::packageVersion("sortable"),
+    system.file("htmlwidgets/plugins/sortable-rstudio", package = "sortable"),
```

Comment:
Might be better to get rid of the `system.file` and specify the `package`, as in https://github.com/rstudio/shiny/blob/250303790c1121a0f8de7bf929e786696e93ab25/R/bootstrap.R#L1548-L1556. I think this will make it play better with shiny prerendered docs.